app/routes/__index/admin/index.tsx

```diff
@@ -0,0 +1,44 @@
+import { Container, Flex, Heading, Input, InputGroup, InputLeftAddon } from '@chakra-ui/react';
+import { FaUsers, FaSearch, FaStickyNote } from 'react-icons/fa';
+import { TbFileCertificate } from 'react-icons/tb';
+import AdminMetricCard from '~/components/admin/admin-metric-card';
+import UsersTable from '~/components/admin/users-table';
+import mockUsers from '~/mock/users';
+
+export default function AdminRoute() {
+  return (
+    <Container maxW="container.xl">
+      <Heading as="h1" size="xl" mt="8">
+        Admin panel
+      </Heading>
+      <Flex flexWrap="wrap">
+        <AdminMetricCard
+          name="Users"
+          tooltipText="Total number of users"
+          value="1923"
```

Comment:
Get these values from the db.

Also, for all the values, format them so they show with commas (e.g., `1,923`):

```ts
new Intl.NumberFormat('en-CA').format(value);
```